toolkits/springernature/packages/springernature-header/HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 
+## 3.1.0 (2022-12-06)
```

Comment:
Happy new year Jon!

Reply:
Wait, you did this in December. I'm awake now

Reply:
> Wait, you did this in December. I'm awake now

Yeah I think it went a bit under the radar end of December